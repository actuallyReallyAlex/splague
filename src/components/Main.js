import React, { Component } from 'react'
import WorldMap from './WorldMap'

class Main extends Component {
  render() {
    return <WorldMap />
  }
}

export default Main
