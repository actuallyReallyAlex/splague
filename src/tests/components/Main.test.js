import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Main from '../../components/Main'

describe('<Main />', () => {
  it('Should render snapshot of Main component.', () => {
    const component = renderer.create(<Main />).toJSON()
    expect(component).toMatchSnapshot()
  })
})
