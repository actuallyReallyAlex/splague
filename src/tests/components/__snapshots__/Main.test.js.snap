// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Should render snapshot of Main component. 1`] = `<Connect(WorldMap) />`;
